en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results.md

```diff
@@ -0,0 +1,112 @@
+
+Key observations:
+
+- More concurrent users mean more requests to the API Manager Gateway. Therefore, the throughput of the API Manager Gateway increases as the number of concurrent users accessing the APIs increases. The maximum throughput is obtained for 200 concurrent users for both “Echo API” and “Mediation API” and the throughput degrades slightly after 200 concurrent users due to resource contentions in the system. The degradation point mainly depends on hardware resources.
+
+- Echo API throughput is much better than the Mediation API. The main reason is that the Mediation API has a mediation extension, which uses a “[Payload Factory](https://docs.wso2.com/display/EI660/PayloadFactory+Mediator)” Mediator. This mediation in the sequence does a JSON to JSON message transformation. That means, the Mediation API reads the message (payload) to do the message transformation and it has a considerable overhead than the “Echo API”, which is similar to a “Direct Proxy”. A “Direct Proxy” does not perform any processing on the messages that pass through it.
+
+The following graph depicts the average response time changes based on the number of concurrent users.
+
+[![Performance result graph - response time]({{base_path}}/assets/img/setup-and-install/performance-result-graph-response-time-0ms-1KiB.png)]({{base_path}}/assets/img/setup-and-install/performance-result-graph-response-time-0ms-1KiB.png)
+
+Key observations:
+
+- The **average response time** increases with the number of concurrent users. As the number of requests to serve increases with more users, there are more resource contentions. Therefore, the number of concurrent users served by the API Gateway needs to be decided on the required response time limits. Again, the average response time also varies as the hardware resources changes. For example, a user can achieve a less response time if the number of CPUs allocated is increased.
+
+- The **mediation API response times** are higher than Echo API due to the performance overhead of mediation extension.
+
+Let’s look at the 90th, 95th, and 99th response time percentiles. This is useful to measure the percentage of requests that exceeded the response time value for a given percentile. A percentile can be used to determine the percentage of requests completed below the particular response time value.
+
+For example, when there are 100 concurrent users, the 90th response time percentile for Echo API is 52ms. This means that 10% of the requests have taken more than 52ms to respond. Similarly, the 99th response time percentile for Echo API is 101ms, which means that 99% of the requests have been completed within 101ms.
+
+[![Performance result graph - response time percentiles]({{base_path}}/assets/img/setup-and-install/performance-result-graph-precentiles-0ms-1KiB.png)]({{base_path}}/assets/img/setup-and-install/performance-result-graph-precentiles-0ms-1KiB.png)
```

Comment:
Images or other files that have caps will not display in Mkdocs based on the existing configurations. Therefore, please rename the image files and update this doc with lower case image file names.

Reply:
Updated via https://github.com/wso2/docs-apim/pull/2017/commits/734ff15a352df15b3cc4cf6cb17f022637ab81ce